TheRickandMorty/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>Rick and Morty API</title>
+</head>
+
+<body>
+    <button id="random" onclick="getCharacters()">Get Info of the Rick and Morty Family</button>
+    <div class="container" id="hide">
+        <div class="card">
```

Comment:
instead of manually writing each card, you can make it so your JS file to do this for you.

inside the for loop you can createNewElement() then append it to container.

this way you will not need to manually add elements in HTML